libs/core/src/lib/time/time.component.ts

```diff
@@ -123,14 +153,72 @@ export class TimeComponent implements OnChanges, ControlValueAccessor {
     /** @hidden */
     setDisabledState(isDisabled: boolean): void {
         this.disabled = isDisabled;
-        this.changeDetRef.detectChanges();
+        this._changeDetRef.detectChanges();
     }
 
+    /** @hidden */
     constructor(
-        public timeI18nLabels: TimeI18nLabels,
-        public timeI18n: TimeI18n,
-        private changeDetRef: ChangeDetectorRef
-    ) {}
+        private _timeI18nLabels: TimeI18nLabels,
+        private _changeDetRef: ChangeDetectorRef
+    ) {
+    }
+
+    /** @hidden */
+    ngOnInit(): void {
+        this._setUpTimeGrid();
+    }
+
+    /** @hidden */
+    handleSecondChange(second: number): void {
+        this.time.second = second;
+        this.onChange(this.time);
+    }
+
+    /** @hidden */
+    handleMinuteChange(minute: number): void {
+        this.time.minute = minute;
+        this.onChange(this.time);
+    }
+
+    /** @hidden */
+    handleNextColumnFocus(column: FdTimeActiveView): void {
```

Comment:
I'd consider getting `TimeColumnComponent` as `ViewChildren` and calling on them `currentTimeColumnComponent.deactivate()`, `nextTimeColumnComponent.activate()` to get rid of this string based approach, but current implementation is also fine.